src/components/ButtonPanel.jsx

```diff
@@ -1,42 +1,39 @@
-import React, { Component } from "react";
+import React from 'react';
+import Button from './Button';
+import './ButtonPanel.css';
 
-class ButtonPanel extends Component {
-  render() {
-    return (
-      <div>
-        <div className="gr-1">
-          <button>AC</button>
-          <button>+/-</button>
-          <button>%</button>
-          <button>/</button>
-        </div>
-        <div className="gr-2">
-          <button>7</button>
-          <button>8</button>
-          <button>9</button>
-          <button>X</button>
-        </div>
-        <div className="gr-3">
-          <button>4</button>
-          <button>5</button>
-          <button>6</button>
-          <button>-</button>
-        </div>
-        <div className="gr-2">
-          <button>1</button>
-          <button>2</button>
-          <button>3</button>
-          <button>+</button>
-        </div>
-        <div className="gr-2">
-          <button>0</button>
-          <button>.</button>
-          <button>=</button>
-          <button> </button>
-        </div>
-      </div>
-    );
-  }
-}
+const ButtonPanel = () => (
+  <div className="button-panel-component">
+    <div className="gr-1">
+      <Button buttonName="AC" />
+      <Button buttonName="+/-" />
+      <Button buttonName="%" />
+      <Button buttonName="/" />
+    </div>
+    <div className="gr-2">
+      <Button buttonName="7" />
+      <Button buttonName="8" />
+      <Button buttonName="9" />
+      <Button buttonName="X" />
+    </div>
+    <div className="gr-3">
+      <Button buttonName="4" />
+      <Button buttonName="5" />
+      <Button buttonName="6" />
+      <Button buttonName="-" />
+    </div>
+    <div className="gr-4">
+      <Button buttonName="1" />
+      <Button buttonName="2" />
+      <Button buttonName="3" />
+      <Button buttonName="+" />
+    </div>
+    <div className="gr-5">
+      <Button buttonName="0" />
+      <Button buttonName="." />
+      <Button buttonName="=" />
+    </div>
```

Comment:
Nice job bringing back these button components

Reply:
But why had you deviated from Milestone 2 of how this component was supposed to be
It also had to send a `color` prop, and some of them a `wide` prop

(i see that you brought them back, it just doesn't show here)